src/main/java/io/jenkins/plugins/coverage/CoveragePublisher.java

```diff
@@ -49,6 +49,8 @@
 
     private String tag;
 
+    private boolean calculateDiffForChangeRequests = true;
```

Comment:
Maybe we should set the default value to false.

Reply:
Agree, thanks